### PR TITLE
Update dashboard-slow-query.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/dashboard-slow-query.md [hunk 1]
```diff
@@ -36,6 +36,12 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
 
 ![显示更多列信息](/media/dashboard/dashboard-slow-queries-list2-v620.png)
 
+### 导出慢查询到本地
```
Comment (lilin90) [inline dashboard/dashboard-slow-query.md:39]:
```suggestion
### 导出慢查询到本地  
```

File: dashboard/dashboard-slow-query.md [hunk 2]
```diff
@@ -36,6 +36,12 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
+
+页面顶部**更多**下可以选择**导出**按钮。点击按钮后，将会把当前列表中的慢查询以 csv 文件形式进行导出。
```
Comment (qiancai) [inline dashboard/dashboard-slow-query.md:41]:
```suggestion
点击页面顶部的 ☰ (**更多**) 按钮可以显示**导出**选项。点击**导出**后，TiDB Dashboard 会将当前列表中的慢查询以 CSV 文件的格式进行导出。
```

File: dashboard/dashboard-slow-query.md [hunk 3]
```diff
@@ -36,6 +36,12 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
 
 ![显示更多列信息](/media/dashboard/dashboard-slow-queries-list2-v620.png)
 
+### 导出慢查询到本地
+
+点击页面顶部 ☰ (**更多**) 可以显示**导出**选项。点击**导出**后，TiDB Dashboard 会将当前列表中的慢查询以 CSV 文件的格式进行导出。
```
Comment (qiancai) [inline dashboard/dashboard-slow-query.md:41]:
```suggestion
点击页面顶部 ☰ (**更多**) 可以显示**导出** (**Export**) 选项。点击**导出** (**Export**) 后，TiDB Dashboard 会将当前列表中的慢查询以 CSV 文件的格式进行导出。
```

**Review by qiancai — APPROVED** [review]
/lgtm

File: dashboard/dashboard-slow-query.md [hunk 4]
```diff
@@ -36,6 +36,12 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
 
 ![显示更多列信息](/media/dashboard/dashboard-slow-queries-list2-v620.png)
 
+### 导出慢查询到本地
+
+点击页面顶部 ☰ (**更多**) 可以显示**导出** (**Export**) 选项。点击**导出** (**Export**) 后，TiDB Dashboard 会将当前列表中的慢查询以 CSV 文件的格式进行导出。
```
Comment (qiancai) [inline dashboard/dashboard-slow-query.md:41]:
```suggestion
点击页面右上角 ☰ (**更多**) 可以显示**导出** (**Export**) 选项。点击**导出** (**Export**) 后，TiDB Dashboard 会将当前列表中的慢查询以 CSV 文件的格式进行导出。
```

**Review by mornyx — APPROVED** [review]

**Pull request merged**